src/ceph-volume/ceph_volume/util/device.py

```diff
@@ -108,8 +108,9 @@ def __init__(self, path, with_lsm=False, lvs=None, lsblk_all=None, all_devices_v
         if os.path.islink(self.path):
             self.symlink = self.path
             real_path = os.path.realpath(self.path)
-            # check if we are not a device mapper
-            if "dm-" not in real_path:
+            lv = lvm.get_single_lv(filters={'lv_path': self.path})
```

Comment:
this implies a new subprocess call in `Device()` that I'd like to avoid

Reply:
Got it. That makes sense.